src/stack/compManager/compManagerProportional/LteCompManagerProportional.cc

```diff
@@ -157,18 +160,20 @@ X2CompProportionalReplyIE* LteCompManagerProportional::buildCoordinatorReply(X2N
     return replyIe;
 }
 
-void LteCompManagerProportional::handleClientRequest(X2CompMsg* compMsg)
+void LteCompManagerProportional::handleClientRequest(inet::IntrusivePtr<X2CompMsg> compMsg)
```

Comment:
Using Ptr<X2CompMsg> allows using both kinds of shared pointers.

Reply:
Was changed to Ptr.